source/Engine.cpp

```diff
@@ -2194,7 +2194,9 @@ void Engine::DoCollisions(Projectile &projectile)
 		// For weapons with a trigger radius, check if any detectable object will set it off.
 		double triggerRadius = weapon.TriggerRadius();
 		if(triggerRadius)
-			for(const Body *body : shipCollisions.Circle(projectile.Position(), triggerRadius))
+		{
+			vector<Body *> circleCollisions;
+			for(const Body *body : shipCollisions.Circle(projectile.Position(), triggerRadius, circleCollisions))
```

Comment:
The variable name circleCollisions sounds a bit too generic. I guess it is possible to use a better name here.